tests/bootstrap.php

```diff
@@ -26,3 +26,6 @@
 
 // Start up the WP testing environment.
 require $_test_root . '/includes/bootstrap.php';
+
+// Load the plugin checker plugin, so the important constants and functions can be used in unit test.
+require_once TESTS_PLUGIN_DIR . '/plugin-check.php';
```

Comment:
@vishalkakadiya Can we remove this, I don't think we need be doing this here as we have other constants defined in this file we can use.

Reply:
@jjgrainger Actually, without loading this plugin file, the `WP_PLUGIN_CHECK_PLUGIN_DIR_PATH` constant is not found while running the `Abstract_PHP_CodeSniffer_Check->run()` function available at [here](https://github.com/10up/plugin-check/blob/trunk/includes/Checker/Checks/Abstract_PHP_CodeSniffer_Check.php#L62) from the test cases. So either we need to change the constant in that `run` method, or I need to load that file in the individual test case. Let me know what's your thoughts here.

Thank you!

Reply:
I agree with @jjgrainger that this shouldn't be necessary, let's find a cleaner solution.

My first question is: Why are these constants not loaded? Shouldn't our plugin be loaded when running unit tests? This suggests there's something wrong in our unit test configuration. I think we are missing something like this in our `bootstrap.php` file: https://github.com/WordPress/performance/blob/1.9.0/tests/bootstrap.php#L35-L38

This is a more appropriate solution as it fixes the root problem.

Reply:
@felixarntz This solution looks good to me. I'm applying this now. Thank you!
CC @jjgrainger